template/koa-rest-api/src/types/jobs.ts

```diff
@@ -11,12 +8,10 @@ export interface Job {
   };
 }
 
-export type JobInput = t.Static<typeof JobInput>;
+export type JobInput = z.infer<typeof JobInput>;
 
-const JobInput = t.Record({
-  hirer: t.Record({
-    id: t.String,
+export const JobInput = z.object({
```

Comment:
```suggestion
export const JobInputSchema = z.object({
```
I would encourage users to use the `Schema` suffix.